src/Client/KeenIOClient.php

```diff
@@ -53,8 +54,8 @@ public static function factory($config = array())
     {
         $default = array(
             'masterKey' => null,
-            'writeKey'  => null,
-            'readKey'   => null,
+            'writeKey' => null,
```

Comment:
In here is what I mean by the array being formatted. This line for example removes the formatting that was applied.